assets/javascripts/admin_assets.js

```diff
@@ -33,15 +33,9 @@ function setupAdminAssets() {
                 makeAssetsByGroup(json);
                 return json.data;
             },
-            error: function(xhr, error, thrown) {
-                var response = xhr.responseJSON;
-                var errorMsg = 'Unable to request asset status: ' +
-                    (response && response.error ? response.error : thrown) +
-                    ' <a class="btn btn-primary" href="javascript: reloadAssetsTable();">Retry</a>';
```

Comment:
Note to self: double-check the importance of `reloadAssetsTable` as opposed to relying on a regular refresh like on other data tables.

Reply:
So what this does is call [table.ajax.reload()](https://datatables.net/reference/api/ajax.reload()) which re-sorts, re-filters and resets pagination. Only the page is not reloaded.
In other words, nothing specific to assets. Ergo we could add this to all tables for consistency.

Reply:
Ok, then add it to all tables.